### PR TITLE
prepping db

### PR DESCRIPTION
**Pull request by huyenminnn**
ready

### CODE REVIEW

**Comment by huyenminnn** [comment]
ready

File: app/Models/Task.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $fillable = ['name'];
```
Comment (quoctd-0607) [inline app/Models/Task.php]: protected $fillable = [
'name',
];

File: app/Models/Task.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    /**
+     * Get the user that owns the task.
+     */
+    public function user()
+    {
+        return $this->belongsTo('App\Models\User');
```
Comment (quoctd-0607) [inline app/Models/Task.php]: return $this->belongsTo(User::class);

File: app/Models/User.php [hunk 3]
```diff
@@ -36,4 +36,12 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    /**
+     * Get all of the tasks for the user.
+     */
+    public function tasks()
+    {
+        return $this->hasMany('App\Models\Task');
```
Comment (quoctd-0607) [inline app/Models/User.php]: tương tự

**Comment by tranducquoc** [comment]
merged

**Pull request merged**